apps/changelog-generator/src/Cli.ts

```diff
@@ -1,8 +1,108 @@
-import * as Command from '@effect/cli/Command';
+import { Command, Options } from '@effect/cli';
+import type { ConfigType, DeploymentConfigV2 } from '@blocksense/config-types';
+import {
+  configDir,
+  readConfig,
+  readEvmDeployment,
+} from '@blocksense/config-types';
+import { Console, Effect } from 'effect';
+import Table from 'cli-table3';
+import type { NetworkName } from '@blocksense/base-utils';
+import { parseNetworkName } from '@blocksense/base-utils';
+import { Terminal } from '@effect/platform';
 
-const command = Command.make('hello');
+const listFeeds = Command.make(
+  'list-feeds',
+  {
+    dir: Options.directory('dir').pipe(Options.withDefault(configDir)),
+  },
+  ({ dir }) =>
+    Effect.gen(function* (_) {
+      const network: NetworkName = yield* _(getNetwork);
+      const config = yield* _(
+        Effect.tryPromise(() => readConfig('feeds_config_v2', dir)),
+      );
+      yield* _(printFeeds(config, network));
+    }),
+);
+
+const command = Command.make('changelog-generator').pipe(
+  Command.withSubcommands([listFeeds]),
+);
 
 export const run = Command.run(command, {
-  name: 'Hello World',
+  name: 'Changelog Generator',
   version: '0.0.0',
 });
+
+const printFeeds = (
+  config: ConfigType<'feeds_config_v2'>,
+  network: NetworkName,
+) =>
+  Effect.gen(function* (_) {
+    const deploymentData = yield* _(getDeploymentData(network));
+    if (!deploymentData) {
+      yield* Console.error(
+        `Deployment data for v2 not found for network: '${network}'`,
+      );
+      return;
+    }
+
+    const table = yield* _(initTable());
```

Comment:
```suggestion
    const table = new Table({
      head: [
        'Feed Name',
        'Address',
        'Decimals',
        'Base Address',
        'Quote Address',
      ],
      style: {
        head: [],
      },
      colWidths: [18, 44, 10, 44, 44],
      wordWrap: true,
    });
```

I don't think we need to wrap each normal function call in `Effect.sync`, so we can inline `initTable`, as I also don't think it would be reused.